notebooks/kubeflow_pipelines/walkthrough/solutions/kfp_walkthrough_vertex_pytorch.ipynb

```diff
@@ -0,0 +1,1517 @@
+{
```

Comment:
<p>* This cell needs pep8 formatting.</p><p>* We should re-use standard classes or functions for the pre-processing in order to keep the training code as short and simple as possible. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>It passes the pep8 check, but I updated it to broke long lines to multiple lines.</p><p> </p><p>In terms of the preprocessing, as I mentioned above, there is no standard handy module to my understanding.</p>